NetCore/Resize.cs

```diff
@@ -48,7 +48,7 @@ public ImageSharpSize ResizeImageSharp()
         {
             using (var image = new ImageSharpImage(Width, Height))
             {
-                image.Mutate(i => i.Resize(ResizedWidth, ResizedHeight));
+                image.Mutate(i => i.Resize(ResizedWidth, ResizedHeight, KnownResamplers.Bicubic));
```

Comment:
Bicubic is the default for ImageSharp, so there is no actual change on this code line. This explains why ImageSharp before/after values remained unchanged.

On the other hand, we need to investigate if the current comparison is fair. If output quality is different and/or not all libraries are capable to handle certain cases, that should be pointed out in benchmark commends or the blog post. For example: do all libraries implement premultiplication to prevent alpha bleeding?

@JimBobSquarePants is planning to have a look soon and come back with more specific details.

Reply:
> Benchmark numbers are not fully meaningful unless you also consider the visual quality of the results, which is harder to quantify.

I couldn't agree more here. Operations should consider far more than just the raw speed. Size, quality, correctness for example. 

Here's a test image which demonstrates issues with several of the libraries. Only MagicScaler, ImageSharp, and System.Drawing get that right.

_Note: I couldn't even get FreeImage to save an input png to jpeg with the current code so I'd consider that useless._ 

**Input**
![kaboom](https://user-images.githubusercontent.com/385879/66915379-9fce3d00-f064-11e9-94ca-933dc4feeed8.png)

**MagicScaler**
![kaboom-MagicScaler](https://user-images.githubusercontent.com/385879/66915290-64cc0980-f064-11e9-8b50-9d0724f382a3.png)

**NetVips**
![kaboom-NetVips](https://user-images.githubusercontent.com/385879/66915292-6564a000-f064-11e9-99bf-e3c1850d23cf.png)

**SkiaSharpBitmap**
![kaboom-SkiaSharpBitmap](https://user-images.githubusercontent.com/385879/66915293-6564a000-f064-11e9-8c21-41f31ca1a00a.png)

**SkiaSharpCanvas**
![kaboom-SkiaSharpCanvas](https://user-images.githubusercontent.com/385879/66915294-65fd3680-f064-11e9-80af-2e8cc7b0cabf.png)

**SystemDrawing**
![kaboom-SystemDrawing](https://user-images.githubusercontent.com/385879/66915296-65fd3680-f064-11e9-84c9-d706d07ae2ed.png)

**ImageSharp**
![kaboom-ImageSharp](https://user-images.githubusercontent.com/385879/66915297-65fd3680-f064-11e9-9d1b-6b454d105ad3.png)

**MagickNET**
![kaboom-MagickNET](https://user-images.githubusercontent.com/385879/66915298-6695cd00-f064-11e9-88f0-cc5fa2fba05b.png)

Reply:
Forgot to say, the System.Drawing code is using some sort of sharpening via `InterpolationMode.HighQualityBicubic`. It also doesn't all subsampling so image quality at 75 will appear sharper than many of the others.